lib/index.js

```diff
@@ -168,12 +171,19 @@ class PluginPassportOAuth {
     return this.getProviderRepository(profile.provider)
       .get(profile._json[this.config.strategies[profile.provider].identifierAttribute])
       .then(userObject => {
-        let req = null;
-
         if (userObject !== null) {
           return Promise.resolve({kuid: userObject.kuid, message: null});
         }
 
+        return Promise.reject({message: `Could not login with strategy "${profile.provider}"`});
```

Comment:
Promise should only reject `Error` objects.

Nitpicking: inside a promise chain, you can throw instead of returning a rejection, for additional readability